web_widget_image_download/static/src/js/web_widget_image_download.js

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright 2016 Flavio Corpa <flavio.corpa@tecnativa.com>
+ * License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+ */
```

Comment:
You can save a couple of LOC removing those dangling comment endings, [like in the template](https://github.com/OCA/maintainer-tools/blob/master/template/module/static/src/js/module_name.js#L1-L2). Not a blocker though.